pyproject.toml

```diff
@@ -28,16 +28,20 @@ types-pyyaml = "^6.0.12.12"
 jsonschema = "^4.2.0"
 types-jsonschema = "^4.2.0"
 prompt-toolkit = "^3.0.39"
-pinecone-text = "^0.7.2"
+pinecone-text = [{version = "^0.7.2"},
+                 {version = "^0.7.2", extras = ["dense"], optional = true}]
+
 tokenizers = "^0.15.0"
-transformers = "^4.35.2"
+transformers = {version = "^4.35.2", optional = true}
```

Comment:
Can we remove this line? seems like it will be taken from pinecone text anyway

Reply:
We need it for `LLamaTokenizer`, which is used with Anyscale's `llama2` LLM. That's why I added an `anyscale` extra.